statsd/client.py

```diff
@@ -13,9 +13,10 @@
 class Timer(object):
     """A context manager/decorator for statsd.timing()."""
 
-    def __init__(self, client, stat, rate=1):
+    def __init__(self, client, stat, tags={}, rate=1):
```

Comment:
Please do not use mutable objects as keyword arguments.
See http://python-guide-pt-br.readthedocs.io/en/latest/writing/gotchas/#mutable-default-arguments